libs/service-portal/core/src/components/GoBack/GoBack.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Button, ResponsiveProp } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { useNavigate } from 'react-router-dom'
+import { m } from '../../'
+import * as styles from './GoBack.css'
+
+type GoBackProps = {
+  display?: ResponsiveProp<
+    'none' | 'flex' | 'block' | 'inline' | 'inlineBlock' | 'inlineFlex'
+  >
+  noUnderline?: boolean
+  truncate?: boolean
+  marginBottom?: ResponsiveProp<
+    | 0
```

Comment:
You could import `ResponsiveSpace` from `'@island.is/island-ui/core'` 

And then do: `marginBottom?: ResponsiveSpace`